packages/devextreme/js/__internal/ui/html_editor/ui/m_aiDialog.ts

```diff
@@ -0,0 +1,207 @@
+import localizationMessage from '@js/common/core/localization/message';
+import $ from '@js/core/renderer';
+import { extend } from '@js/core/utils/extend';
+import type { ToolbarItem } from '@js/ui/popup';
+import SelectBox from '@js/ui/select_box';
+import TextArea from '@js/ui/text_area';
+
+import type { CommandDefinition, CommandsMap } from '../utils/m_ai';
+import DialogBase from './m_baseDialog';
+
+const AI_DIALOG_COMMANDS_WITH_OPTIONS = ['translate', 'changeStyle', 'changeTone'];
+
+export interface AIDialogShowPayload {
+  currentCommand: string;
+  currentOption?: string;
```

Comment:
[naming]
I suggest u to try to make this and related names more precise. Now it's a bit difficult for me to read the code because `option` word is widely used across our code base but with another meaning
```suggestion
  currentCommandOptions?: string;
```